antora-playbook.yml

```diff
@@ -10,7 +10,15 @@ site:
   title: Couchbase Docs
   url: https://docs.couchbase.com
   start_page: home::index.adoc
-  robots: allow
+  robots: |
+    User-agent: *
+    Allow: /sdk-api/*-client/
+    Allow: /sdk-api/couchbase-core-io/
+    Allow: /sdk-api/couchbase-transactions-java/
+    Allow: /sdk-api/couchbase-transactions-cxx/
```

Comment:
Once the C++ client becomes GA, this separate repo can also be hidden from indexing.